e2e/helpers/api_helper.js

```diff
@@ -109,6 +109,14 @@ const grantCaseAccess = async (caseId, user, role) => {
   return await post(url, data, headers);
 };
 
+const revokeCaseAccess = async (caseId, user, role) => {
+  const authToken = await getAuthToken();
+  const url = `${config.fplServiceUrl}/testing-support/case/${caseId}/revoke-access`;
+  const headers = getHeaders(authToken);
+  const data = { email: user.email, password: user.password, role: role };
+  return await post(url, data, headers);
+};
+
```

Comment:
Why do we need this for document removal? Should we separate it to another PR / ticket?

Reply:
Yes, can extract it to another ticket.